app/views/devise/shared/_registration_form.html.slim

```diff
@@ -1,7 +1,26 @@
 = form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f|
-  form.form-horizontal[novalidate]
+  form.form-horizontal
     .row
       = devise_error_messages!
+      / ------for the future implementation------
```

Comment:
this kind of future features better to comment as `# TODO: implement with Issue#123`